src/mapReducers.js

```diff
@@ -1,4 +1,5 @@
 import _ from 'lodash';
+import { isTargetAllAction } from './context';
```

Comment:
It is responsibility of keySelector function to return ALL based on internal logic of action creator or key selector function. I agree about groups, maybe we could allow key selector to return arrays of keys also.